pkg/platform/aws/aws.go

```diff
@@ -105,6 +107,10 @@ func NewConfig() *config {
 	}
 }
 
+func (c *config) clusterDomain() string {
+	return fmt.Sprintf("%s.%s", c.ClusterName, c.DNSZone)
```

Comment:
Does this not take into account the variable `cluster_domain_suffix`? What if user provides some other `cluster_domain_suffix`? 

Correct me if I am wrong, I am just trying to identify potential discrepancy between variables.

Reply:
I don't think `cluster_domain_suffix` is needed here

@invidian

Reply:
`cluster_domain_suffix` is for cluster internal DNS, no?

Reply:
yes.